atomicapp/utils.py

```diff
@@ -377,6 +377,53 @@ def rm_dir(directory):
         distutils.dir_util.remove_tree(directory)
 
     @staticmethod
+    def getUid(user):
+        """
+        Get the UID of the specific user by grepping /etc/passwd unless
+        we are in a container.
+
+        Returns:
+            (int): User UID
+        """
+
+        # If we're in a container we should be looking in the /host/ directory
+        if Utils.inContainer():
+            passwd = os.path.join(HOST_DIR, "etc/passwd")
+        else:
+            passwd = "/etc/passwd"
+
+        uidmap = {}
+        for line in file(passwd):
+            lst = line.split(':')
+            uidmap[lst[0]] = lst[2]
+
+        return int(uidmap[user])
+
+    @staticmethod
+    def setFilePermissions(src):
```

Comment:
so this name is a bit of a misleading name. we are not actually changing any file perms, just changing who owns the file - so maybe we should call it setFileOwnerGroup()